keyboards/iris/keymaps/saviof/keymap.c

```diff
@@ -0,0 +1,110 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJ 3
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJ,
+};
+
+//Tap Dance Declarations
+enum {
+  TD_SPC_ENT = 0
+};
+
+//Tap Dance Definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_SPC_ENT] = ACTION_TAP_DANCE_DOUBLE(KC_SPC, KC_ENT)
+};
+
+#define KC_LOWR LT(_LOWER, KC_LEFT) //LOWER
```

Comment:
You need to add `#define KC_ KC_TRNS` above Line 27.

It was only the `#define _______ KC_TRNS` that needed deletion before.